src/transport/fusedev/linux_session.rs

```diff
@@ -21,6 +21,7 @@ use nix::errno::Errno;
 use nix::fcntl::{fcntl, FcntlArg, OFlag};
 use nix::mount::{mount, umount2, MntFlags, MsFlags};
 use nix::poll::{poll, PollFd, PollFlags};
+use nix::sys::epoll::{epoll_ctl, EpollOp, EpollEvent, EpollFlags};
```

Comment:
Is `nix::sys::epoll` an wrapper for `kqueue`?

Reply:
No, it's epoll only.